.github/workflows/main.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Lint
         run: yarn lint
 
-      - name: Test
-        run: yarn test --ci --coverage --maxWorkers=2
+      # - name: Test
+      # run: yarn test --ci --coverage --maxWorkers=2
```

Comment:
There aren't any tests so this causes the workflow to fail. Didn't see any simple tests to quickly add, since all the functions are calling either the Sanity client or Smartling.

Reply:
Yes -- I think it would be nice to add some mocks for external services, might require a light refactor of how we're fetching.

Reply:
Agreed